projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -109,6 +110,11 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     this.caseEditDataService.caseEditForm$.subscribe({
       next: editForm => this.editForm = editForm
     });
+    this.caseEditDataService.caseIsLinkedCasesJourneyAtFinalStep$.subscribe({
+      next: isLinkedCasesJourneyAtFinalStep => {
```

Comment:
@johnbenjamin-hmcts - this can be rewritten as a single line without the curly braces

Reply:
@Daniel-Lam - I had it as one line first, but then used an additional console.log statement and then missed reverting it to one line. I have updated the code now.